automation/project/Settings.scala

```diff
@@ -41,7 +41,7 @@ object Settings {
   val commonSettings =
     commonBuildSettings ++ testSettings ++ List(
     organization  := "org.broadinstitute.dsde.firecloud",
-    scalaVersion  := "2.11.8",
+    scalaVersion  := "2.12.4",
```

Comment:
Scalatest 3.0.5 (latest version) compatible with Scala 2.12.4
https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.12/3.0.5/scalatest_2.12-3.0.5.pom

Reply:
```import sbt.Keys.scalaVersion
import sbt._

object Dependencies {
  val jacksonV = "2.9.2"
  val akkaV = "2.5.7"
  val akkaHttpV = "10.1.1"
  val scalaV = scalaVersion.toString()

  val serviceTestV = "0.9-e094fde"
  val workbenchModelV  = "0.11-2ce3359"
  val workbenchMetricsV  = "0.3-c5b80d2"
  val workbenchGoogleV = "0.16-c5b80d2"

  val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
  val workbenchMetrics: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-metrics" % workbenchMetricsV

  val workbenchExclusions = Seq(
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-model_$scalaV"),
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-util_$scalaV"),
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-metrics_$scalaV")
  )```

Reply:
@rtitle Can I get your opinion on Andrew's comment?
couple questions I have are:
-  Scala versions for workbench-model, workbench-util and workbench-metrics are all different and different from that in Firecloud-ui/automation project. Is that a problem to use `val scalaV = scalaVersion.toString()` in exclusionRule?
-  Is common to use `.toString` in sbt Settting.scala file?

Reply:
I agree with Andrew's original comment that the 2.12 versions of the workbench-libs libraries must be excluded now that 2.12 is being used. But I think `scalaVersion` is incorrect in this case because it includes the minor version (e.g. 2.12.6) whereas the libraries just have the major version (2.12). The `scalaBinaryVersion` variable might be what you want here. So:

```
val workbenchExclusions = Seq(
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-model_${scalaBinaryVersion.value}"),
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-util_${scalaBinaryVersion.value}"),
    ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = s"workbench-metrics_${scalaBinaryVersion.value}")
)
```

Does that work? (I haven't tested it)

Reply:
cool. thanks. I didn't know about `scalaBinaryVersion` key before. `.value` cannot be used, so it is `val excludeV = scalaBinaryVersion` to compile okay